scripts/local_testnet/start_local_testnet.sh

```diff
@@ -75,9 +74,11 @@ else
     echo "Not rebuilding Lighthouse Docker image."
 fi
 
-# Stop local testnet
-kurtosis enclave rm -f $ENCLAVE_NAME 2>/dev/null || true
+if [ "$KEEP_ENCLAVE" = false ]; then
+  # Stop local testnet
+  kurtosis enclave rm -f $ENCLAVE_NAME 2>/dev/null || true
+fi
 
-kurtosis run --enclave $ENCLAVE_NAME github.com/ethpandaops/ethereum-package --args-file $NETWORK_PARAMS_FILE
+kurtosis run --enclave $ENCLAVE_NAME github.com/ethpandaops/ethereum-package --image-download always --args-file $NETWORK_PARAMS_FILE
```

Comment:
NOTE: This allows kurtosis to evaluate the file against existing enclave and make only necessary changes.

Reply:
Oops I might be wrong about this. Just ran into some issues locally but didn't have time to investigate. I'll test again tomorrow before we merge this.

Reply:
Ok so it turns out that `-k` and `image-download=always` may not be compatible.

Because if we re-run a testnet with `image-download=always`, it pulls new image and re-creates all containers (we lose all beacon chain data), and none of the LH node is able to sync from genesis.

However they do work independently and are both useful for the purpose of local testing. I think it's just an edge case we need to keep in mind.

TL;DR:
- we can modify network params, log levels, add additional services and re-run the testnet without having to destroy and start from scratch, using the `-k` option of this script.
- `--image-download=always` will make sure we're always testing using the latest image (e.g. geth) and not testing against outdated ones.

Thanks @chong-he for help with testing!

Reply:
I didn't fully understand. If we rerun with `-k` option and `image-download=always` erases the beacon chain data, then wouldn't we need an additional option to toggle `image-download=always`?

Reply:
Yes you are right, I thought about this but didn't bother with it because we use `lighthouse:local` by default and it wouldn't trigger a container recreation, but if user switches to use `sigp/lighthouse` then it would be an issue - maybe I should only include `image-download=always` when `-k` is not present, would this be better? or should we introduce another option? I prefer the former, because If user wants to update any of the images, they probably wouldn't use `-k`. (I misunderstood the behaviour earlier!)

Unrelated note: The original thinking for this script was to support the common use cases of non kurtosis/ existing users, and avoid turning this into a monster script - I'm still hoping we could keep this small and simple. For more advance users they would potentially prefer to use `kurtosis` directly for more flexibility.

Reply:
Yeah maybe we can just drop `image-download=always` if `-k` isn't present. That seems like a better option if we don't want to add unnecessarily to the script.

Reply:
Done!